### ISSUE TITLE
question about get best modle

### ISSUE THREAD

**Issue by bighawkin** [CaiLiLab/MuRaL#7]
The run example is : get_best_mural_models ./ray_results/your_experiment_name/Train_*/progress.csv
question1 ：But when mural_train parameter “n_trials” is more than one ，how to judge the best model. 

another question 2：what is the best  parameter. I notice the model example of human n_trials is 10, so I follow it. Is it best to set it this way？

another question 3：About “Scaling MuRaL-predicted mutation rates to per base per generation rates”，Can my final result in this part be to obtain the mutation rate of per base across the genome？

Thank you very much！

**Comment by cailigd** [comment]
Re #1: The output of 'get_best_mural_models' has the best model (the one with lowest validation loss) for each trial. For the whole experiment, you can choose the model with lowest loss across all trials based on the output of  'get_best_mural_models'.

Re #2: Generally, the number of trials to run depends on the training data size and computation resource.  With large training data, the best models of different trials usually look quite similar, so don't need to run many trials to get a reasonably good model. As larger training data or more trials will take longer time,  if the computation resource is limited or the training data size is large, it is recommended to run fewer trials (e.g. 3 trials) to reduce the training time.  

Re #3: No sure about the meaning of your question. If you need per base per generation rates for each base, you need to do the scaling part as the documentation tells. The original rates generated by mural_train/mural_predict can be considered as relative mutation rates within a specific class of sites (e.g. A/T sites or C/G sites).

**Comment by bighawkin** [comment]
Thanks for your reply. I want to know if the scaling part can cover the entire genome from beginning to end, and if it is computationally feasible. Also, is the strategy of splitting each chromosome for training reliable?  （ get modle of each chromosome and combined the training model results of each chromosome to analyze the high mutation rate regions of the entire genome）
